frontend/packages/local-storage-operator-plugin/src/components/disks-list/ocs-disks-list.tsx

```diff
@@ -0,0 +1,270 @@
+  switch (status) {
+    case 'Not Ready':
+      return 'Preparing to replace';
+    case 'Ready':
+      return 'Replacement ready';
+    case 'Failed':
+      return 'Replacement failed';
+    default:
+      return null;
+  }
+};
+
+const getAlertsBasedStatus = (alertName: string): OCSDiskStatus => {
+  switch (alertName) {
+    case OSD_DOWN_ALERT:
+      return 'Not responding';
+    case OSD_DOWN_AND_OUT_ALERT:
+      return 'Offline';
+    default:
+      return null;
+  }
+};
+
+const getTiResource = (ocsState: OCSColumnState): WatchK8sResource => {
+  /* Template Instance to be watched when replacement is triggered */
+  const templateName = `${OSD_REMOVAL_TEMPLATE}-${ocsState.replacingDisk.osd}`;
+  return ocsState.replacingDisk.disk
+    ? {
+        kind: TemplateInstanceModel.kind,
+        name: templateName,
+        namespace: CEPH_STORAGE_NAMESPACE,
+      }
+    : null;
+};
+
+export const tableColumnClasses = [
+  '',
+  '',
+  cx('pf-m-hidden', 'pf-m-visible-on-xl'),
+  cx('pf-m-hidden', 'pf-m-visible-on-2xl'),
+  cx('pf-m-hidden', 'pf-m-visible-on-lg'),
+  cx('pf-m-hidden', 'pf-m-visible-on-xl'),
+  Kebab.columnClass,
+];
+
+const diskHeader = () => [
+  {
+    title: 'Name',
+    sortField: 'path',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[0] },
+  },
+  {
+    title: 'Disk State',
+    sortField: 'status.state',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[1] },
+  },
+  {
+    title: 'OCS Status',
+    sortField: '',
+    transforms: [],
+    props: { className: tableColumnClasses[1] },
+  },
+  {
+    title: 'Type',
+    sortField: 'type',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[2] },
+  },
+  {
+    title: 'Model',
+    sortField: 'model',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[3] },
+  },
+  {
+    title: 'Capacity',
+    sortField: 'size',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[4] },
+  },
+  {
+    title: 'Filesystem',
+    sortField: 'fstype',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[5] },
+  },
+  {
+    title: '',
+    sortField: '',
+    transforms: [],
+    props: { className: tableColumnClasses[6] },
+  },
+];
+
+const diskRow: RowFunction<DiskMetadata, OCSMetadata> = ({
+  obj,
+  index,
+  key,
+  style,
+  customData,
+}) => {
+  const { ocsState, dispatch } = customData;
+  const diskName = obj.path;
+  return (
+    <TableRow id={obj.deviceID} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>{obj.path}</TableData>
+      <TableData className={tableColumnClasses[1]}>{obj.status.state}</TableData>
+      <OCSStatus ocsState={ocsState} diskName={diskName} className={tableColumnClasses[1]} />
+      <TableData className={tableColumnClasses[2]}>{obj.type || '-'}</TableData>
+      <TableData className={cx(tableColumnClasses[3], 'co-break-word')}>
```

Comment:
```suggestion
      <TableData className={tableColumnClasses[3]}>
```
Put in the tableColumnClasses[3] itself.